changelogs/fragments/491-win_pssession_configuration.yml

```
@@ -0,0 +1,4 @@
+bugfixes:
+- win_pssession_configuration - Fix parser error (Invalid JSON primitive: icrosoft.WSMan.Management.WSManConfigContainerElement)
```

This will be an invalid changelog framework due to the inner `: `. You need to quote this value to ensure it stays a string

```suggestion
- 'win_pssession_configuration - Fix parser error (Invalid JSON primitive: icrosoft.WSMan.Management.WSManConfigContainerElement)'
```